traits/servicemonitor/README.md

```diff
@@ -0,0 +1,62 @@
+# ServiceMonitor Trait
+
+oam-kubernetes-runtime supports existing third-party Kubernetes resources, this section introduce how to use k8s CR [ServiceMonitor](https://github.com/coreos/kube-prometheus#defining-the-servicemonitor-for-each-additional-namespace) in oam-kubernetes-runtime. 
+
+## Pre-requisites
+
+- Please follow [common prerequisites](../../README.md) to install Crossplane and OAM.
+
+- Deploy Prometheus and Grafana using the Prometheus Operator ([kube-prometheus](https://github.com/coreos/kube-prometheus)).
+```
+git clone https://github.com/coreos/kube-prometheus.git
```

Comment:
Do they offer a helm chart or something like that? Clone code is a bit complex.

Reply:
Yes, it can deploy by helm charts. 
I also find that we can let grafana discovery dashboard data automatically and I'm working to add it for the demo. So I'll refine they together later.